interfaces/Portal/src/app/pages/login/login.page.ts

```diff
@@ -46,13 +51,27 @@ export class LoginPage implements OnDestroy {
 
   private msalSubscription: Subscription;
 
+  public ssoUserIsNotFound: boolean;
+  public isPopupBlocked = false;
+
   constructor(
     private authService: AuthService,
     private translate: TranslateService,
     private router: Router,
     private msalService?: MsalService,
   ) {}
 
+  ngOnInit(): void {
+    // Retrieve the error message from session storage
+    this.ssoUserIsNotFound =
+      sessionStorage.getItem(SSO_ERROR_KEY) === SSO_ERRORS.notFound;
+    sessionStorage.removeItem(SSO_ERROR_KEY);
+
+    if (isIframed()) {
+      this.isPopupBlocked = isPopupBlocked();
+    }
```

Comment:
Oh, forgot to mention previously:

This should all be in an if-block for `environment.use_sso_azure_entra`, so it doesn't have to be run when SSO is not enabled.

```suggestion
    if (environment.use_sso_azure_entra) {
      this.ssoUserIsNotFound =
        sessionStorage.getItem(SSO_ERROR_KEY) === SSO_ERRORS.notFound;
      sessionStorage.removeItem(SSO_ERROR_KEY);

      if (isIframed()) {
        this.isPopupBlocked = isPopupBlocked();
      }
    }
```

Reply:
ESLint is fixed too.